### PR TITLE
adding attribute 'assignee' & adding 'resolved' in attr_reader for faults api

### PR DESCRIPTION
**Pull request by ajish**

### CODE REVIEW

File: lib/honeybadger-api/fault.rb [hunk 1]
```diff
@@ -16,7 +17,8 @@ def initialize(opts)
         @klass = opts[:klass]
         @action = opts[:action]
         @component = opts[:component]
-        @message = opts[:message]
+        @message = opts[:message],
+        @assignee = opts[:assignee],
```
Comment (murraysum) [inline lib/honeybadger-api/fault.rb]: @ajish Extra commas aren't required. Please can you remove? :+1:

File: lib/honeybadger-api/fault.rb [hunk 2]
```diff
@@ -2,8 +2,9 @@ module Honeybadger
   module Api
     class Fault
 
-      attr_reader :id, :project_id, :klass, :action, :component, :message,
-        :environment, :notices_count, :comments_count, :last_notice_at, :created_at
+      attr_reader :id, :project_id, :klass, :action, :component, :message, :assignee,
+        :environment, :notices_count, :comments_count, :last_notice_at, :created_at, :ignored,
+        :resolved
```
Comment (murraysum) [inline lib/honeybadger-api/fault.rb]: @ajish Can you update the spec with the appropriate missing tests for `assignee`, `ignored` and `resolved` please?

File: lib/honeybadger-api/fault.rb (hunk 1 above)
Comment (murraysum) [inline lib/honeybadger-api/fault.rb]: @ajish I haven't had a chance to look at the returned JSON from honeybadger but what does the `assignee` JSON look like? Is it something like:

```json
  "assignee": {
    "email": "westley@honeybadger.io", 
    "name": "Westley"
  }
```

If so then you will want to build out a new `User` object. We do this in other parts of the gem. Better to keep this consistent with the rest of the gem in my opinion. :+1:

**Comment by murraysum** [comment]
@ajish Thanks for submitting some changes. I've taken a look and noticed some issues that you'll need to address. Give me a shout when you want me to take another look :+1:

**Comment by ajish** [comment]
sure, I will try and get this done in the next 24 hours.

On Tue, Jul 7, 2015 at 3:21 AM, Murray Summers <notifications@github.com>
wrote:

> @ajish <https://github.com/ajish> Thanks for submitting some changes.
> I've taken a look and noticed some issues that you'll need to address. Give
> me a shout when you want me to take another look [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/murraysum/honeybadger-api/pull/5#issuecomment-119007376>
> .
>



-- 
Ajish Balakrishnan
+91-8197068965

   - in.linkedin.com/pub/ajish-balakrishnan/1a/214/ba7/

**Comment by murraysum** [comment]
@ajish Hey, I haven't heard from you in a while and this fell off my radar. 

In the latest v2.0.0 release I have now included assignee attribute for faults.

I didn't realise at the time but you can tell if a fault is resolved using the `resolved?` method which I think is clearer that an `attr_reader` of `resolved`.

I am going to close this pull request as I think all the behaviour you need is now present. Please let me know if it isn't. 👍

**Pull request closed without merging**